components/MapsDedi/index.vue

```diff
@@ -59,6 +63,28 @@ export default {
     isMitra () {
       return this.joinSelected === 'Mitra'
     }
+  },
+  async mounted () {
+    const resLocation = await this.$axios.get('/villages/list-with-location')
+    console.log(resLocation.data.data)
```

Comment:
Minor: 

Hello, kak @adzharamrullah. I think `console.log` is unnecessary code kak because it is used to debugging in development mode, right?

Reply:
ok thanks doohan, i forgot to remove that, i will fix that

Reply:
@doohanas i think you wrong review changes, that changes is not already in last commit

Reply:
sorry kak 😄 @adzharamrullah